homeassistant/components/bmw_connected_drive/strings.json

```diff
@@ -124,7 +124,14 @@
         "name": "Remaining fuel percent"
       },
       "climate_status": {
-        "name": "Climate status"
+        "name": "Climate status",
+        "state": {
+          "cooling": "Cooling",
+          "heating": "Heating",
+          "inactive": "Inactive",
+          "standby": "Standby",
+          "unknown": "Unknown"
```

Comment:
If the state is unknown, the snesor should return `None` since HA will register that as `Unknown`

Reply:
ok, thanks, pushed now, hope it makes sense